internal/query/projection/settings_relational.go

```diff
@@ -0,0 +1,1768 @@
+				},
+				{
+					Event:  instance.LabelPolicyIconAddedEventType,
+					Reduce: s.reduceIconAdded,
+				},
+				{
+					Event:  instance.LabelPolicyIconRemovedEventType,
+					Reduce: s.reduceIconRemoved,
+				},
+				{
+					Event:  instance.LabelPolicyLogoDarkAddedEventType,
+					Reduce: s.reduceLabelLogoAdded,
+				},
+				{
+					Event:  instance.LabelPolicyLogoDarkRemovedEventType,
+					Reduce: s.reduceLogoRemoved,
+				},
+				{
+					Event:  instance.LabelPolicyIconDarkAddedEventType,
+					Reduce: s.reduceIconAdded,
+				},
+				{
+					Event:  instance.LabelPolicyIconDarkRemovedEventType,
+					Reduce: s.reduceIconRemoved,
+				},
+				{
+					Event:  instance.LabelPolicyFontAddedEventType,
+					Reduce: s.reduceFontAdded,
+				},
+				{
+					Event:  instance.LabelPolicyFontRemovedEventType,
+					Reduce: s.reduceFontRemoved,
+				},
+				// 		// Password Complexity
+				{
+					Event:  instance.PasswordComplexityPolicyAddedEventType,
+					Reduce: s.reducePassedComplexityAdded,
+				},
+				{
+					Event:  instance.PasswordComplexityPolicyChangedEventType,
+					Reduce: s.reducePasswordComplexityChanged,
+				},
+				// 		// Password Policy
+				{
+					Event:  instance.PasswordAgePolicyAddedEventType,
+					Reduce: s.reducePasswordPolicyAdded,
+				},
+				{
+					Event:  instance.PasswordAgePolicyChangedEventType,
+					Reduce: s.reducePasswordPolicyChanged,
+				},
+				// 		// Lockout Policy
+				{
+					Event:  instance.LockoutPolicyAddedEventType,
+					Reduce: s.reduceLockoutPolicyAdded,
+				},
+				{
+					Event:  instance.LockoutPolicyChangedEventType,
+					Reduce: s.reduceLockoutPolicyChanged,
+				},
+				// 		// Domain Policy
+				{
+					Event:  instance.DomainPolicyAddedEventType,
+					Reduce: s.reduceDomainPolicyAdded,
+				},
+				{
+					Event:  instance.DomainPolicyChangedEventType,
+					Reduce: s.reduceDomainPolicyChanged,
+				},
+				// 		// Security Policy
+				{
+					Event:  instance.SecurityPolicySetEventType,
+					Reduce: s.reduceSecurityPolicySet,
+				},
+			},
+		},
+	}
+}
+
+func (s *settingsRelationalProjection) reduceLoginPolicyAdded(event eventstore.Event) (*handler.Statement, error) {
+	var orgId *string
+	var policyEvent policy.LoginPolicyAddedEvent
+	var ownerType domain.OwnerType
+	switch e := event.(type) {
+	case *instance.LoginPolicyAddedEvent:
+		policyEvent = e.LoginPolicyAddedEvent
+		ownerType = domain.OwnerTypeInstance
+	case *org.LoginPolicyAddedEvent:
+		policyEvent = e.LoginPolicyAddedEvent
+		ownerType = domain.OwnerTypeOrganization
+		orgId = &policyEvent.Aggregate().ResourceOwner
+	default:
+		return nil, zerrors.ThrowInvalidArgumentf(nil, "HANDL-YYPxS", "reduce.wrong.event.type %v", []eventstore.EventType{org.LoginPolicyAddedEventType, instance.LoginPolicyAddedEventType})
+	}
+
+	return handler.NewStatement(&policyEvent, func(ctx context.Context, ex handler.Executer, projectionName string) error {
+		settingJSON, err := json.Marshal(policyEvent)
```

Comment:
Shouldn't we fill in the information into a struct defined in the relational package?
That way we avoid unmarshalling problems.